internal/serverinstall/docker.go

```diff
@@ -95,6 +96,28 @@ func (i *DockerInstaller) Install(
 		s.Update("Detected existing Waypoint server.")
 		s.Status(terminal.StatusWarn)
 		s.Done()
+
+		for _, container := range containers {
+			if container.State == "running" {
+				s = sg.Add("Existing Waypoint server container is already running.")
```

Comment:
I don't think we need to output anything for this?

Reply:
True, we do say it was detected above this. I thought it might be nice to say for sure that it's running but it might be redundant :smile:

Reply:
Yeah that was my thought: I guess I assume it'd be running if an installation was detected.